tasks/setup-Debian.yml

```diff
@@ -12,3 +12,10 @@
     repo: 'deb https://artifacts.elastic.co/packages/5.x/apt stable main'
     state: present
     update_cache: yes
+
+- name: Install nodejs and friends for elasticdump
```

Comment:
Added the nodejs + npm install to setup-RedHat, so all of this should make sense now.  Haven't had a chance to test it in Redhat/CentOS, though...